Fix performance-history-scripting.md Example Scripts

comnam90 opened the pull request:
Updated example script for `Sample 3: Noisy neighbor? That's write!` as the current implementation didn't work.

 * Moved `Format-Iops` function into the context of the invoke-command so that it could actually be called on the remote machine.    
 * Added an `if` statement to the `do` loop in `Format-Iops` so that if the initial raw value is less than 1000, it doesn't end up being output as `0 K`.    
 * Changed values passed to `Format-Iops` in the `PsCustomObject` block to actually be raw values rather than full `MSCluster_HealthMetric` objects.    
 * Changed `Sort-Object` statement in final output to be descending to actually output the Top 10 rather than the Bottom 10.

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/windowsserverdocs?pullRequest=1084) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/MicrosoftDocs/windowsserverdocs?pullRequest=1084) before we can accept your contribution.<br/><br/>:x: comnam90 [sign now](https://cla.opensource.microsoft.com/MicrosoftDocs/windowsserverdocs?pullRequest=1084)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/MicrosoftDocs/windowsserverdocs?pullRequest=1084) it.</sub>

comnam90 commented:
Updated example script for `Sample 6: Memory hog, you can run but you can't hide` as the current implementation didn't work.

 * Moved `Format-Bytes` function into the context of the invoke-command so that it could actually be called on the remote machine.    
 * Added an `if` statement to the `do` loop in `Format-Bytes` so that if the initial raw value is less than 1024, it doesn't end up being output as `0 KB`.    
 * Changed values passed to `Format-Bytes` in the `PsCustomObject` block to actually be raw values rather than full `MSCluster_HealthMetric` objects.    
 * Changed `Sort-Object` statement in final output to be descending to actually output the Top 10 rather than the Bottom 10.

cosmosdarwin commented:
Changes look good to me. Thanks very much, Ben!
